### ISSUE TITLE
some bugs

### ISSUE THREAD

**Issue by Felix0805** [heshenghuan/LSTM-NER#12]
I think there are some bugs about FLAGS

**Comment by Felix0805** [comment]
There are many UnrecognizedFlagErrors

**Comment by heshenghuan** [comment]
@Felix0805 
Thank you for reporting the bugs. 
Can you please provide some details about the bugs?

**Comment by Felix0805** [comment]
When you run the whole project
In main.py line 245-247 have the error:
absl.flags._exceptions.UnrecognizedFlagError: Unknown command line flag 'label2idx'

**Comment by heshenghuan** [comment]
Do you have the Traceback for this bug?

'label2idx' isn't an argument, you don't need to give a 'label2idx' in command line.

**Comment by Felix0805** [comment]
yeah,I don't give a 'label2idx' in command line, but 'label2idx' is undefined in Flag

**Comment by heshenghuan** [comment]
OK, I understood. It looks like the API has changed. And it was not recommended to use FLAGS like that.

Thank you for your advice. I think it can be fixed by using a dictionary to store those undefined variables.
